source/help/getting-started/signing-in.rst

```diff
@@ -4,8 +4,9 @@ Signing in
 --------------
 
 You can sign in to your team from the root page of your Mattermost site
-``https://domain.com``. After signing in, you will be able to view a
-list of your teams and select which one to open.
+``https://domain.com``. After signing in, the team that appears first on your
```

Comment:
Should it be "You can sign in to your team from the root page of your Mattermost site, ``https://<domain>.com``" instead?

There, `<domain>` would be inserted by the user,